### ISSUE TITLE
get logos/commitments from OT users and implementations

### ISSUE THREAD

**Issue by bensigelman** [opentracing/opentracing.io#60]
It will help OpenTracing if we have early commitments from tracing systems and tracing users. Starting with projects that have people actively contributing to OpenTracing (in alphabetical order):
- [x] Appdash (@slimsag / @bg451)
- [ ] AppNeta (@dkuebric)
- [ ] CockroachDB (@tschottdorf)
- [x] LightStep (@bensigelman)
- [ ] Uber (@yurishkuro)

Others are welcome to add other tracing projects and companies/organizations that want to use OT. I have a bunch of other ideas here but will need to send some emails first.

We'll check the box for the project once we're explicitly clear that it's fine to use their logo on opentracing.io.

**Comment by adriancole** [comment]
TL;DR an organization like zipkin can't be said to be OpenTracing, just
because I have participated. I appreciate the value of marketing, and know
the intent is good, just I feel uncomfortable about this.

Zipkin is primarily a backend, as opposed to a tracer. Ex most tracers
arent in the Zipkin org and they are self-managed. So far, none are using
OT apis. Closest is spring-cloud-sleuth, but that's not a zipkin project
and so far the apis don't match up.. Basically it is under the influence,
but not an OT api either, and I don't see an immediate path of convergence
(maybe not for months).

Zipkin has an empty repo waiting for a python OT tracer. It has been
untouched with no code for a month. I would think this would have activity
before we could go so far as to link zipkin as an org to OT.
https://github.com/openzipkin/zipkin-python-opentracing

Wrt linking people based on where they come from, I think it is more
typical to ask folks. For example, I formerly was listed as from Pivotal,
which was fair enough. However, tagging me as Zipkin is more interesting.

On the one hand, I have certainly raised issues about how things impact
zipkin, but also Sleuth, and mostly out of interest for the
distributed-tracing workgroup. Similarly, Mick is working on Cassandra
tracing, too. My main stick is I don't want to conflate my concerns for
zipkin and OT as speaking for others in zipkin. I'm perhaps more careful
about this than others might be.

If might be best to let work speak for itself. Ex when an OT zipkin tracer
occurs, note and link to that. So far, there's no operable OT zipkin tracer
I'm aware of, but if there was commitment by someone to do that, it would
make sense to link them.

End long-winded 2p!

**Comment by bensigelman** [comment]
@adriancole this won't be nearly as long as your message, but it must be said: you *do* have an important (even "Pivotal", har har har) role in Zipkin as a project. The idea here would be something like "Zipkin thinks OpenTracing is a good idea and would like to interoperate with it", not "Zipkin has already fully integrated with OpenTracing". I.e., more an expression of intent.

And, on that note, an endorsement for OpenTracing from the lead Zipkin maintainer means a great deal. In fact, I think it means a lot more than someone writing an OT-Zipkin binding library... anyone can do that if they so choose but it wouldn't actually *mean* anything about the intended strategy of Zipkin as a project.

**Comment by adriancole** [comment]
Interesting lemme think about this.

**Comment by adriancole** [comment]
TL;DR; please just remove me. I just want to be who I am, which is a person who contributes as they have time.

I don't have strong enough feelings towards decisions made to make a persistent endorsement, knowing I don't have time or energy to drive things. I need to spend more time on Zipkin and the reality is time on OT usually doesn't benefit the needs Zipkin tracer folks raise to me (except yuri). Regardless, I don't feel comfortable conflating my personal circumstances with the "opinion of zipkin"

**Comment by slimsag** [comment]
I can of course see both sides of the discussion here. I won't pretend that I know what goes into the management of a much larger project like Zipkin, nor will I pretend that I know the legal aspect of some of the larger projects on this list.

I will say, however, that I think seeing that these people are interested in Zipkin, that there is a community of individuals who are actively trying to get tracing systems more compatible with one another under one roof, is a good thing. There is value in showing that to outsiders, the people who will be running into tracing systems for the first time but don't actively crawl mailing lists or issue trackers like this one. This isn't a rebuttle against any of the points that Adrian raised above, though. I've only observed good opinions towards opentracing thus far, but Appdash is also a much smaller community and project overall than Zipkin is I'm sure.

Maybe for all the others here who run into the no-single-leader (in the case of Zipkin) or legal troubles (in the case of other companies here), it might make sense to make this more of a:

```
## People who think OpenTracing is cool / a good idea

- Adrian Cole, a contributor to Zikpin.
- Stephen Gutekanst, primary maintainer of Appdash.
- Yuri Shkuro, developer at Uber.
...
```

obviously, with better titles and wording. Not trying to stir up any dust here, just spit-balling ideas!

**TL;DR:**

I think it's a good idea, feel totally free to list Appdash in a "we think this is a good idea" kind of mindset. The various logos you'd probably want to use are here: [logo_red.png](https://github.com/sourcegraph/appdash-data/blob/master/data/logo_red.png), [logo_white.png](https://github.com/sourcegraph/appdash-data/blob/master/data/logo_white.png), and the icon [red_spinner.png](https://github.com/sourcegraph/appdash-data/blob/master/data/red_spinner.png). ofc. you're welcome to use the Sourcegraph logo too, but I think it'd be less appropriate given the context / since we've not yet used OpenTracing for Sourcegraph code.

**Comment by bensigelman** [comment]
@slimsag, all makes sense... and I agree, the Appdash logo is more appropriate here. thanks!
